include/png.h

```diff
@@ -1,9 +1,9 @@
 
```

Comment:
All the png and zlib header files were re-copied straight from fltk-1.4.0rc1/png and fltk-1.4.0rc1/zlib.